forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -25,13 +25,15 @@ import com.joinforage.forage.android.ui.ForagePINEditText
  *
  * * [Tokenizing card information][tokenizeCard]
  * * [Checking the balance of a card][checkBalance]
- * * [Collecting a customer's PIN for a payment and
+ * * [Collecting a card PIN for a payment and
  * deferring the capture of the payment to the server][deferPaymentCapture]
  * * [Capturing a payment immediately][capturePayment]
- * * [Refunding a payment][refundPayment]
+ * * [Collecting a customer's card PIN for a refund and defer the completion of the refund to the
+ * server][deferPaymentRefund]
+ * * [Refunding a payment immediately][refundPayment]
  *
  * @param posTerminalId **Required**. A string that uniquely identifies the POS Terminal
- * used for a transaction.
+ * used for a transaction. The max length of the strength is 255 characters.
```

Comment:
```suggestion
 * used for a transaction. The max length of the string is 255 characters.
```

Reply:
Oh my goodness @djoksimo This is the funniest typo/Freudian slip ever. Moving too quickly this morning. Thank you!!